source/administration/changelog.md

```diff
@@ -188,6 +188,9 @@ Multiple setting options were added to `config.json`. Below is a list of the add
 
 Mattermost v5.19.0 contains low to high level security fixes. [Upgrading](http://docs.mattermost.com/administration/upgrade.html) is recommended. Details will be posted on our [security updates page](https://about.mattermost.com/security-updates/) 30 days after release as per the [Mattermost Responsible Disclosure Policy](https://www.mattermost.org/responsible-disclosure-policy/).
 
+- **v5.19.2, release day TBD**
+  - Fixing an issue with a data race on post properties. [MM-21378](https://mattermost.atlassian.net/browse/MM-21378)
```

Comment:
This may need more details.

Reply:
I'd probably add the same extract from https://github.com/mattermost/docs/pull/3451. 
```
Direct access to the ``Props`` field in the ``model.Post`` structure has been deprecated. The available ``GetProps()`` and ``SetProps()`` methods should now be used. Also, direct copy of the ``model.Post`` structure must be avoided in favor of the provided ``Clone()`` method.
```